test/acceptance/support/processor.js

```diff
@@ -0,0 +1,22 @@
+const azureClient = require('./azure-client')
+const path = require('path')
+
+const uploadFile = async () => {
+  const HTTP_OK = 201
```

Comment:
`200` is `OK` whereas `201` is `Created`.  Suggest renaming to `HTTP_CREATED` so not misleading.